agent/provider/tests/keystore_cli.rs

```diff
@@ -180,7 +180,7 @@ fn test_keystore_add_certificate_permissions(
     command.arg("--json").assert().success();
 
     let result = list_certificates_command(&cert_dir).unwrap();
-
+    println!("Result: {result:?}");
```

Comment:
Please add task for outbound 2.0 about adding keystore CLI tests for golem certs

Reply:
Created task https://github.com/golemfactory/yagna/issues/2483